hoomd/Communicator.cc

```diff
@@ -2724,6 +2797,32 @@ void Communicator::beginUpdateGhosts(uint64_t timestep)
                 }
             }
 
+        if (flags[comm_flag::angmom])
+            {
+            ArrayHandle<Scalar4> h_angmom(m_pdata->getAngularMomentumArray(),
+                                          access_location::host,
+                                          access_mode::read);
+            ArrayHandle<Scalar4> h_angmom_copybuf(m_angmom_copybuf,
+                                                  access_location::host,
+                                                  access_mode::overwrite);
+            ArrayHandle<unsigned int> h_copy_ghosts(m_copy_ghosts[dir],
+                                                    access_location::host,
+                                                    access_mode::read);
+            ArrayHandle<unsigned int> h_rtag(m_pdata->getRTags(),
+                                             access_location::host,
+                                             access_mode::read);
+
+            // copy velocity of ghost particles
```

Comment:
```suggestion
            // copy angmom of ghost particles
```
Should this be angmom?